semgrep-core/Core/Rule.ml

```diff
@@ -44,7 +44,7 @@ type xlang =
 
 type regexp = string * Pcre.regexp
 let pp_regexp fmt (s, _) =
-  Format.fprintf fmt "%s" s
+  Format.fprintf fmt "\"%s\"" s
```

Comment:
I usually use `%S`, which prints a double-quoted ocaml string literal. It escapes a bunch of characters too in addition to escaping `"`.
```
Format.fprintf fmt "%S" s
```

Reply:
Good to know, will fix.